registry/sql-registry/main.py

```diff
@@ -74,4 +78,39 @@ def get_feature_lineage(feature: str) -> dict:
     return lineage.to_dict()
 
 
-app.include_router(prefix = rp, router=router)
+@router.post("/projects")
+def new_project(definition: dict) -> UUID:
+    id = registry.create_project(ProjectDef(**to_snake(definition)))
+    return {"guid": str(id)}
+
+
+@router.post("/projects/{project}/datasources")
+def new_project_datasource(project: str, definition: dict) -> UUID:
```

Comment:
>UUID [](http://example.com/codeflow?start=62&length=4)

def new_project_datasource(project: str, definition: dict) -> UUID:
    project_id = registry.get_entity_id(project)
    id = registry.create_project_datasource(project_id, SourceDef(**to_snake(definition)))
    return {"guid": str(id)}

I guess you meant return json, not UUID? Same applies to other post methods

Reply:
Fixed